README.md

```diff
@@ -16,6 +16,7 @@ Read more about the Badger DAO and its community at https://badger.com/dao-and-c
 The recommended installation tool for this repository is [`poetry`](https://python-poetry.org/docs/):
 ```
 poetry install
+git submodule update --init --recursive
```

Comment:
worth seeing the progress to understand if its getting hang out or not, since that repo takes usually quite a while to clone

```suggestion
git submodule update --init --recursive --progress
```